src/components/PlayButton/PlayButton.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+    ButtonHTMLAttributes,
+    DetailedHTMLProps,
+    HTMLInputTypeAttribute,
+    ReactNode,
+} from "react";
+import { FaPlay, FaPause, FaStop } from "react-icons/fa";
+import style from "./PlayButton.module.scss";
+
+type PlayButtonTypeAttribute = "play" | "pause" | "stop" | "disabled";
```

Comment:
Change to ButtonType = keyof typeof buttonVariants

Reply:
That either creates a cyclic reference, or forces me to change the type of playButtonVariants to something less meaningful, I prefer this approach, but I would like to know if you can think of any way (setting playButtonVariants key type to string and setting it as const didn't work)